WorkFlowManagerFrontend/src/app/organizations/organization-details/organization-details.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container">
+  <div *ngIf="organization">
+    <h1>Organization: {{organization.name}}</h1>
+    <p *ngIf="organization.description">Description: {{organization.description}}</p>
+  </div>
+  <div class="panel-container">
+    <div class="users-panel">
+      <div>
+        <mat-form-field class="profile-field">
+          <mat-label>Search user</mat-label>
+          <input matInput type="text" [(ngModel)]="searchUser" placeholder="Search user"/>
+          <mat-icon matSuffix>search</mat-icon>
+        </mat-form-field>
+      </div>
+  
+      <button class="add-btn" mat-flat-button color="primary" (click)="openAddUserDialg()">
+        <span>Add user</span>
+        <mat-icon matSuffix>add</mat-icon>
+      </button>
+  
+      <mat-list class="users-list" *ngIf="members$ | async as members">
+        <mat-list-item *ngFor="let member of members | filter: searchUser">
+          <span matListItemTitle>{{member.name}}</span>
+        </mat-list-item>
+      </mat-list>
+    </div>
+  
+    <div class="roles-panel">
+      <div>
+        <mat-form-field>
+          <mat-label>Search role</mat-label>
+          <input matInput type="text" [(ngModel)]="searchUser" placeholder="Search role"/>
+          <mat-icon matSuffix>search</mat-icon>
+        </mat-form-field>
+      </div>
+  
+      <button class="add-btn" mat-flat-button color="primary">
+        <span>Create role</span>
+        <mat-icon matSuffix>add</mat-icon>
+      </button>
+     
+      <mat-list class="roles-list">
+        <mat-list-item *ngFor="let role of roles">
+          <span matListItemTitle>{{role.name}}</span>
+          <button mat-button></button>
+          <!-- <button mat-button [matMenuTriggerFor]="menu">
```

Comment:
@rxgxs21, sprawdź czy ten kawałek kodu jest potrzebny, bo nie wiem co to robi. Ten `[matMenuTriggerFor]="menu"` nie pozwala mi uruchomić aplikacji xD